packages/bigquery/src/index.js

```diff
@@ -90,6 +90,15 @@ util.inherits(BigQuery, common.Service);
  *
  * @example
  * bigquery.createDataset('my-dataset', function(err, dataset, apiResponse) {});
+ *
+ * //-
+ * // If the callback is omitted, we'll return a Promise.
+ * //-
+ * bigquery.createDataset('my-dataset')
```

Comment:
Is there a system for when `.then` goes on a new line? I think we should always put it on a new line for consistency.